### PR TITLE
Allow treating a missing highlight language as error

### PR DESCRIPTION
**Pull request by SamWilsn**
This is a tiny non-breaking feature. I already did the work before seeing the note about your forum in the pull request template. I hope I'm not being too rude by skipping making a post there :sweat_smile: 

Sanity check:

* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/getzola/zola/pulls) for the same update/change?

## Code changes
(Delete or ignore this section for documentation changes)

* [x] Are you doing the PR on the `next` branch?

If the change is a new feature or adding to/changing an existing one:

* [x] Have you created/updated the relevant documentation page(s)?

### CODE REVIEW

File: components/config/src/config/markup.rs [hunk 1]
```diff
@@ -27,6 +41,8 @@ pub struct ThemeCss {
 pub struct Markdown {
     /// Whether to highlight all code blocks found in markdown files. Defaults to false
     pub highlight_code: bool,
+    /// Emit either "error" or "warn" for missing highlight languages. Defaults to "warn".
+    pub highlight_missing_level: HighlightLanguageLevel,
```
Comment (Keats) [inline components/config/src/config/markup.rs]: I'm wondering if we can just do `error_on_unknown_highlight`. Do we have anything other than error or warn?

Comment (SamWilsn) [reply components/config/src/config/markup.rs]: I was going for symmetry with `internal_level` in the link checker. I don't have a strong preference.

I suppose we could also have a "do nothing" level, if users are annoyed by the warnings.

Comment (Keats) [reply components/config/src/config/markup.rs]: I wouldn't hide the warnings in any way. Let's go with a bool for now

Comment (SamWilsn) [reply components/config/src/config/markup.rs]: Done!

**Review by Keats — APPROVED** [review]
Thanks!

**Comment by pddiffer053-alt** [comment]
Aboutme

**Pull request merged**